contrib/auto-render/auto-render.js

```diff
@@ -1,6 +1,6 @@
 /* eslint no-console:0 */
 
-import katex from "katex";
+import katex from "../../katex";
```

Comment:
Can you run `npm run build` and see if this ends up including katex within the built file?

Reply:
Yes, you are right. With that change `katex` is included in the `auto-render` build.
I tried leaving `katex` absolute imported here and included in the `test/auto-render-spec` build, but still check fails.
I should confess I have no idea how to fix this. :disappointed:.

Reply:
Moreover, unfortunately, the check doesn't fail locally. So I find a bit difficult to test potential fixes.

Reply:
Hi @kevinbarabash.

Finally I managed to fix the check failure.
`auto-render` is built without including `katex`. During test execution, since we are in a `node.js` environment, it tries to import `katex`as `node.js` module but no such module can be found in the default paths.
My solution consists in adding the root folder to the `NODE_PATH` environment variable before executing `jest`.